save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/usersettings/UserSettingsOrganizationsMenuView.kt

```diff
@@ -26,6 +27,25 @@ import react.dom.html.ReactHTML.ul
 @Suppress("MISSING_KDOC_TOP_LEVEL", "TOO_LONG_FUNCTION", "LongMethod")
 class UserSettingsOrganizationsMenuView : UserSettingsView() {
     private val organizationListCard = cardComponent(isBordered = false, hasBg = true)
+    private val comparator: Comparator<OrganizationWithUsers> =
+            compareBy<OrganizationWithUsers> { orderedOrganizationStatus[it.organization.status] }
+                .thenBy { it.organization.name }
+
+    /**
+     * This function delete [organizationWithUsers] by [selfOrganizationWithUserList],
```

Comment:
Please proofread the KDoc.

Still not clear enough.

Reply:
Done [13603f1](https://github.com/saveourtool/save-cloud/pull/1546/commits/13603f16b4f13604e28a152d433dc1ade897cb6b)